src/chopper_layout.cpp

```diff
@@ -16,79 +16,132 @@ void set_up_subparser_layout(seqan3::argument_parser & parser, chopper::layout::
+                      "(see option --rearrange-user-bins).",
                       seqan3::option_spec::standard,
                       seqan3::arithmetic_range_validator{static_cast<size_t>(1), std::numeric_limits<size_t>::max()});
 
-    parser.add_flag(config.estimate_union, 'u', "estimate-union",
-                    "[HLL] Estimate the union of kmer sets to possibly improve the binning. Only possible if the "
-                    "directory [INPUT-PREFIX]_sketches is present.");
-
-    parser.add_flag(config.rearrange_bins, 'r', "rearrange-bins",
-                    "[HLL] Do a rearrangement of the bins which takes into account similarity. Enabling this option "
-                    "also enables -u. Only possible if the directory [INPUT-PREFIX]_sketches is present.");
-
-    parser.add_section("Special options");
-    parser.add_line("\n");
-
-    parser.add_flag(config.determine_num_bins, '\0', "determine-num-bins",
-                    "If given, the programm will determine the best number of technical bins by "
-                    "doing multiple binning runs. The -b option will then be an upper bound.");
-
-    parser.add_flag(config.force_all_binnings, '\0', "force-all-binnings",
-                    "If given together with --determine-num-bins, all binnings up to the chosen t_max are computed "
-                    "instead of stopping when the expected query costs become worse.");
+    parser.add_subsection("HyperLogLog Sketches");
+    parser.add_line("To improve the layout, you can estimate the sequence similarities using HyperLogLog sketches.");
+
+    parser.add_flag(config.estimate_union,
+                    '\0', "estimate-union",
+                    "Use sketches to estimate the sequence similarity of a set of user bins. This will improve the "
+                    "layout computation as merging user bins that do not increase technical bin sizes will be "
+                    "preferred. Attention: Only possible if the directory [INPUT-PREFIX]_sketches is present.");
+
+    parser.add_flag(config.rearrange_bins,
+                    '\0', "rearrange-user-bins",
+                    "As a preprocessing step, rearranging the order of the given user bins based on their sequence "
+                    "similarity may lead to favourable small unions and thus a smaller index."
+                    "Attenation: Also enables --estimate-union and is only possible if the directory "
+                    "[INPUT-PREFIX]_sketches is present.");
+
+    parser.add_subsection("Parameter Tweaking:");
+    // -----------------------------------------------------------------------------------------------------------------
+    parser.add_option(config.alpha,
+                      '\0', "alpha",
+                      "The layout algorithm optimizes the space consumption of the resulting HIBF but currently has no "
+                      "means of optimizing the runtime for querying such an HIBF. In general, the ratio of merged bins "
```

Comment:
Also here, we do include the expected amount of queries, no?

Reply:
we only track it in the statistics but the DP algorithm does not include it.